src/test/java/org/jenkinsci/plugins/durabletask/WindowsBatchScriptTest.java

```diff
@@ -132,4 +133,35 @@ private void testWithPath(String path) throws Exception {
         c.cleanup(ws);
     }
 
+    @Test public void scriptAffectedByLineEndings() throws Exception {
+        char[] pad254 = new char[254];
+        char[] pad231 = new char[231];
+        Arrays.fill(pad254, '#');
+        Arrays.fill(pad231, '#');
+        // script is found on https://www.dostips.com/forum/viewtopic.php?f=3&t=8988#p58860
+        String script =
+            "@echo off\n" +
+            "goto :zwei\n" +
+            ":" + new String(pad254) + "\n" +
+            ":" + new String(pad254) + "\n" +
+            ":" + new String(pad254) + "\n" +
+            ":" + new String(pad231) + "\n" +
+            "\n" +
+            ":eins\n" +
+            "ECHO eins\n" +
+            "goto :eof\n" +
+            "\n" +
+            ":zwei\n" +
+            "echo zwei\n" +
+            "goto :eins\n";
+        Controller c = new WindowsBatchScript(script).launch(new EnvVars(), ws, launcher, listener);
+        while (c.exitStatus(ws, launcher, listener) == null) {
+            Thread.sleep(100);
+        }
+        ByteArrayOutputStream baos = new ByteArrayOutputStream();
+        c.writeLog(ws, baos);
+        assertEquals(0, c.exitStatus(ws, launcher, listener).intValue());
```

Comment:
What happens before the fix? Just a non-zero exit code here?

Reply:
It sets exit code to 1 and prints error message that cmd can't find first label (`eins`).
I believe we shouldn't check the message since it's locale-specific.
```bat
> cmd /c call test-crlf.bat
zwei
The system cannot find the batch label specified - eins

> echo %errorlevel%
1
```